src/lightning/pytorch/trainer/trainer.py

```diff
@@ -278,6 +279,10 @@ def __init__(
 
             inference_mode: Whether to use :func:`torch.inference_mode` or :func:`torch.no_grad` during
                 evaluation (``validate``/``test``/``predict``).
+
+            barebones: Whether to run in "barebones mode", where all features that could impact performance are
```

Comment:
can we actually call it "speed" or something similar here? The term performance might be confusing as it could also related to predictive performance (aka "accuracy")

Reply:
Used "raw speed" but happy to change it